sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -89,6 +89,25 @@ def _env_var_or_default(var_name, default_val=""):
 # TODO: Enabled when duplicate logging issue is solved
 # _EXPORTER_DIAGNOSTICS_ENABLED = _is_exporter_diagnostics_enabled()
 
+# Instrumentations
+
+# Opt-out
+_AZURE_SDK_INSTRUMENTATION_NAME = "azure_sdk"
+_FULLY_SUPPORTED_INSTRUMENTED_LIBRARIES = (
+    "django",
+    "fastapi",
+    "flask",
+    "psycopg2",
+    "requests",
+    "urllib",
+    "urllib3",
+    _AZURE_SDK_INSTRUMENTATION_NAME,
```

Comment:
Nit: Does lint prevent you from putting this name first?

Reply:
No. Done.